### ISSUE TITLE
Add [CLS] and [SEP] tokens in Usage

### ISSUE THREAD

**Issue by tomohideshibata** [huggingface/pytorch-pretrained-BERT#176]
Thank you for this great job.

In the Usage section, the `[CLS]` and `[SEP]` tokens should be added in the beginning and ending of  `tokenized_text`?

```
# Tokenized input
text = "Who was Jim Henson ? Jim Henson was a puppeteer"
tokenized_text = tokenizer.tokenize(text)
```

In the current example, if the first token is masked (this position should be reserved for `[CLS]`), the result will be strange.

Thanks.

**Comment by thomwolf** [comment]
You are right, I'll fix the readme

**Comment by hughperkins** [comment]
So, just to clarify, I should add '[CLS]' and '[SEP]' to the beginning and end of each utterance respectively, and it's a bug in the examples that they dont do this?

**Comment by wahlforss** [comment]
@hughperkins did you get any clarification on this?
